packages/server/src/lib/targeting.ts

```diff
@@ -106,6 +107,20 @@ export const correctSignedInStatus = (
     }
 };
 
+export const hasConsentedStatus = (
+    hasConsented: boolean,
+    consentStatus?: ConsentStatus,
+): boolean => {
+    switch (consentStatus) {
+        case 'HasConsented':
+            return hasConsented === true;
+        case 'HasNotConsented':
+            return hasConsented === false;
+        default:
```

Comment:
so if the test targets 'Everyone' we return false?

Reply:
good spot - this should be true, as we want to target everyone regardless of their consent

Reply:
might be worth adding some test cases to `targeting.test.ts`

Reply:
Thanks, we've updated the case statement and added tests